src/components/Reports/GoalCalculator/SummaryReport/Steps/PresentingYourGoalStep/PresentingYourGoal.test.tsx

```diff
@@ -0,0 +1,175 @@
+    ...OriginalModule,
+    ResponsiveContainer: ({ children }) => (
+      <OriginalModule.ResponsiveContainer width={800} height={800}>
+        {children}
+      </OriginalModule.ResponsiveContainer>
+    ),
+  };
+});
+
+jest.mock('src/hooks/useOrganizationId', () => ({
+  useOrganizationId: jest.fn(() => 'organization-id-1'),
+}));
+
+jest.mock('src/hooks/useAccountListId', () => ({
+  useAccountListId: jest.fn(() => 'account-list-id-1'),
+}));
+
+const TestComponent: React.FC = () => (
+  <SnackbarProvider>
+    <ThemeProvider theme={theme}>
+      <GqlMockedProvider<{
+        GetUser: GetUserQuery;
+        GetUsersOrganizationsAccounts: GetUsersOrganizationsAccountsQuery;
+        GetAccountList: GetAccountListQuery;
+        GetOrganizations: GetOrganizationsQuery;
+      }>
+        mocks={{
+          GetUser: {
+            user: {
+              id: 'account-list-id-1',
+              firstName: 'Obiwan',
+              lastName: 'Kenobi',
+            },
+          },
+          GetUsersOrganizationsAccounts: {
+            userOrganizationAccounts: [
+              {
+                id: 'user-org-account-id-1',
+                username: 'obiwan.kenobi',
+                latestDonationDate: '2023-12-01',
+                lastDownloadedAt: '2023-12-15',
+                organization: {
+                  id: 'organization-id-1',
+                  name: 'Cru - United States of America',
+                  apiClass: 'DataSync',
+                  oauth: false,
+                },
+              },
+            ],
+          },
+          GetAccountList: {
+            accountList: {
+              receivedPledges: 100,
+            },
+          },
+          GetOrganizations: {
+            organizations: [
+              {
+                id: 'organization-id-1',
+                organizationType: 'Cru',
+              },
+            ],
+          },
+        }}
+      >
+        <GoalCalculatorProvider>
+          <PresentingYourGoal />
+        </GoalCalculatorProvider>
+      </GqlMockedProvider>
+    </ThemeProvider>
+  </SnackbarProvider>
+);
+
+describe('PresentingYourGoal', () => {
+  beforeEach(() => {
+    beforeTestResizeObserver();
+  });
+
+  it('renders cell text and table headings', async () => {
+    const { getByRole } = render(<TestComponent />);
+    expect(
+      getByRole('heading', { name: 'Personal Information' }),
+    ).toBeInTheDocument();
+    expect(getByRole('cell', { name: 'User' })).toBeInTheDocument();
+    expect(getByRole('cell', { name: 'Mission Agency' })).toBeInTheDocument();
+    expect(
+      getByRole('heading', { name: 'Monthly Support Needs' }),
+    ).toBeInTheDocument();
+
+    await waitFor(() => {
+      expect(getByRole('cell', { name: 'Obiwan Kenobi' })).toBeInTheDocument();
+      expect(
+        getByRole('cell', { name: 'Cru - United States of America' }),
+      ).toBeInTheDocument();
+      expect(getByRole('cell', { name: 'Orlando, FL' })).toBeInTheDocument();
+    });
```

Comment:
This `waitFor`+`getByRole` combo can be simplified to `await findByRole`.

```suggestion
    expect(await findByRole('cell', { name: 'Obiwan Kenobi' })).toBeInTheDocument();
    expect(
      getByRole('cell', { name: 'Cru - United States of America' }),
    ).toBeInTheDocument();
    expect(getByRole('cell', { name: 'Orlando, FL' })).toBeInTheDocument();
```